django/pyrog/api/serializers/mapping.py

```diff
@@ -0,0 +1,97 @@
+"""Serializer for the full mapping."""
```

Comment:
Why aren't they the same as in import_export.py?

Reply:
because we didn't succeed using the import_export ModelSerializer-s (which make db calls) in river-api since pyrog doesn't share its model with river-api. But we need those Serializers (basic ones not ModelSerializer) in River and Pagai.

Reply:
Can't wait for that to happen 😅